R/run_prep_calculation_loans.R

```diff
@@ -326,6 +331,92 @@ run_prep_calculation_loans <- function(input_path_project_specific,
       credit = credit_type
     )
 
+  # join AR PAMS data to obtain information on emission factors
+  loans_results_company <- loans_results_company %>%
+    dplyr::inner_join(
+      production_forecast_data %>% dplyr::distinct(.data$company_id, .data$name_company),
+      by = c("company_name" = "name_company")
+    ) %>%
+    dplyr::select(-.data$id) %>%
+    dplyr::rename(id = .data$company_id)
+
+  # This aggregates AR PAMS to company_technology level to get emission factors
+  # Note that this is done including all assets, i.e. we look at global
+  # emission factors here
+  # TODO: run for each scenario geography
```

Comment:
what are the consequences of not having this todo implemented?

Reply:
emissions factors can differ a bit on the regional level, as they are a weighted mean of the underlying technology types. If in one region different ways to extract oil are used compared to another region, the EFs may differ. This will not be large scale differences, but it would be more precise providing these on a regional level taking into account the local production mix

Reply:
Ideally this would be done when decoupling from pacta, as a similar aggregation will have to be done for the production data as well.